dashboard/client/src/pages/serve/ServeApplicationDetailPage.tsx

```diff
@@ -118,7 +118,7 @@ export const ServeApplicationDetailPage = () => {
                     ? `Application config for ${application.name}`
                     : `Application config`
                 }
-                code={application.deployed_app_config}
+                code={application.deployed_app_config || "null"}
```

Comment:
what happens if we don't do this change?

Reply:
This will cause an error, since `code` is used in <CodeDialogButton> as:

```
 <Typography className={classes.configText}>
              {typeof code === "string" ? code : yaml.dump(code, { indent: 2 })}
</Typography>

```
yaml.dump function expects its first argument to be an object or an array.

Reply:
can we instead just print out a "-" instead of rendering `CodeDialogButton` when the value is null?